src/main/scala/DemoFortis.scala

```diff
@@ -134,7 +137,7 @@ object DemoFortis {
           .map(post => {
             val source = post.post.getPermalinkUrl.toString
             val language = languageDetection.detectLanguage(post.post.getMessage)
-            val analysis = Analysis(language = language)
+            val analysis = Analysis(language = language, keywords = keywordExtractor.extractKeywords(post.post.getMessage))
```

Comment:
Makes sense. The other thing is we should be filtering the results based on the presence of keywords. I just put this here as a quick POC.